prebid-mobile/adops-line-item-setup-adform.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: Setup Line Items for Adform
+description: Setup line items for Adform
+pid: 2
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-adops
+---
+
+
+<div class="bs-docs-section" markdown="1">
+
+# Step by Step Line Item Setup for Adform
+
+* TOC
+{:toc }
+
+This page describes step by step how to set up Prebid Mobile line items for Adform to serve ads on app with the Prebid SDK.
+
+
+## Step 1. Create a Campaign
+
+- Create a client under which you would like to run in-app HB campaign (1)
+- Select the longest possible period (2).
+- Set goal to **Unlimited** (3).
+
+{: .pb-med-img :}
+  ![Adform Campaign Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform1.png "Example Adform Campaign Setup")
+
+## Step 2. Create a Line Item
+
+In the General settings Details section:
+- Select delivery type: **Non-guaranteed** (1)
+- Select status: **Booked** (2)
+
+{: .pb-med-img :}
+  ![Adform Line Item Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform2.png "Example Adform Line Item")
```

Comment:
thank you so much for adding screen tips to the images! It would be nice if you could make each one a little different. Like "Line Item General Settings," "Line Item Timing," etc.